docs/zh-cn/dev/user_doc/registry_spi.md

```diff
@@ -20,7 +19,7 @@ registry.servers|127.0.0.1:2181|zk 连接地址
 #### FAQ
 1: not found registry plugin
```

Comment:
If you change the English version, please remember to modify the Chinese version.